build.gradle

```diff
@@ -481,9 +483,12 @@ applicationDefaultJvmArgs = [
   "java.base/java.nio=ALL-UNNAMED"
 ]
 
+
 run {
   args project.hasProperty("besu.run.args") ? project.property("besu.run.args").toString().split("\\s+") : []
   doFirst {
+    applicationDefaultJvmArgs.add(0, "-Dotel.resource.attributes=service.name=besu-dev")
+    applicationDefaultJvmArgs.add(0, "-javaagent:"+ configurations.javaAgent.singleFile.toPath() + "")
```

Comment:
What is the overhead cost of adding this agent and not using opentelemetry?  My concern is I don't want to impose a universal performance penalty.

Reply:
Perhaps a separate `besu-tracing` start script could address the agent overhead for non-tracing instances?
https://stackoverflow.com/a/59724219/8020

Reply:
:+1: Sounds good to me. I'll address that in a bit.

OK with using tracing with `./gradlew run` by default?

Reply:
Sure.  I don't know of anyone who needs raw performance from a local build.

Reply:
Talked some more on rocket.chat - I am offering that we use an env var "TRACING", that if set, will trigger tracing to be enabled. I am afraid otherwise of explaining away why we have 2 executable scripts to the casual user.